test/util.py

```diff
@@ -0,0 +1,9 @@
+from mycroft.configuration.config import LocalConf, DEFAULT_CONFIG
+from copy import copy
+
+__config = LocalConf(DEFAULT_CONFIG)
+
+
+# Base config to use when mocking
+def base_config():
+    return copy(__config)
```

Comment:
Should this be `deepcopy` so that merged attribute of sub elements don't mutate the original?

Reply:
If you think so it probably should, my python knowledge breaks down around here :) Will fix

Reply:
Only difference is:
```python
a = {'1': {'2': 'original'}}
b = copy(a)
c = deepcopy(a)

a['1']['2'] = 'altered'

print(b['1']['2'])  # altered
print(c['1']['2'])  # original
```

Reply:
Thanks.